File: Basic Coding Problems/01 Fundamentals/009 Write a program to enter temperature in Fahrenheit and convert it into Celsius/C/temperature_conversion.c
#include <stdio.h>

int main() {
    float fahrenheit, celsius;

    printf("Enter the temperature in Fahrenheit: ");
    scanf("%f", &fahrenheit);

    celsius = (fahrenheit - 32) * 5 / 9;

    printf("The temperature in Celsius is: %.2f\n", celsius);

    return 0;
}

File: Basic Coding Problems/01 Fundamentals/009 Write a program to enter temperature in Fahrenheit and convert it into Celsius/Java/temperature_conversion.java
import java.util.Scanner;

public class temperature_conversion {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter the temperature in Fahrenheit: ");
        float fahrenheit = scanner.nextFloat();

        float celsius = (fahrenheit - 32) * 5 / 9;

        System.out.println("The temperature in Celsius is: " + celsius);
    }
}